### ISSUE TITLE
Cannot use snippets in automations, these work elsewhere in Budibase

### ISSUE THREAD

**Issue by ConorWebb96** [Budibase/budibase#13590]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
Self
- Method: docker compose
- Budibase Version: 2.24.0
- App Version: 2.24.0

**Describe the bug**
Snippets don't seem to work in automations

**To Reproduce**
1. Create a snippet in the design or data screen
2. Use it in either of these screens, see that it works
3. Go to the automation tab and use a snippet inside it
4. See that this doesn't work here.

**Expected behavior**
Snippet functions should be able to be used within automations. (Anywhere that has a bindable field, should allow js snippets to be used.)

**Additional Notes**
It might be a good idea to allow snippets to be used within the JS scripting step within Automations as having these reusable functions may come in handy here.

**Comment by aptkingston** [comment]
Just confirming that this issue is an enhancement to add snippets to automations, right? I deliberately didn't include snippets there since automations do so many unique things with bindings. If you were able to see snippets somewhere in bindings and they just didn't work then that's definitely a bug, because you shouldn't be able to see them anywhere.

Otherwise I'm just gonna remove the bug label from this since this is as intended - automations should not have snippets right now.

Can't remember those details off the top of my head from our chat about this one.

**Comment by ConorWebb96** [comment]
Yes, Sorry selected the wrong template which auto adds the bug tag!

**Comment by ConorWebb96** [comment]
If this is an enhancement, it might be a good idea to remove snippets from being able to be selected in automation.
